src/lib/connection/connector/postmessage-legacy.js

```diff
@@ -116,7 +119,7 @@ const sendMessage = store => async (cmd, ...payload) => {
     case actions.nativeUpload:
       return sendInvocationCall(invocationMapping[cmd], payload);
     default:
-      throw new Error('Command ' + cmd + ' not supported by driver');
+      return sendValue(fallBackSendMessage.apply(null, [cmd, ...payload]));
```

Comment:
should we give the developer a hint about the degraded/fallback mode and at least add a debug message?

Reply:
Debug out is already in there to hint for what is used iirc